rust/main/utils/run-locally/src/invariants/termination_invariants.rs

```diff
@@ -127,17 +129,24 @@ pub fn relayer_termination_invariants_met(
     let total_tx_id_log_count = *log_counts
         .get(&tx_id_indexing_line_filter)
         .expect("Failed to get tx id indexing log count");
+
+    // there are 3 txid-indexed events:
```

Comment:
this comment block should probably go above the assert with `total_tx_id_log_count as u64 >= expected_td_id_logs,`